system-addon/content-src/components/CollapsibleSection/CollapsibleSection.jsx

```diff
@@ -89,6 +89,39 @@ class Info extends React.PureComponent {
 
 const InfoIntl = injectIntl(Info);
 
+class Disclaimer extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.onAcknowledge = this.onAcknowledge.bind(this);
+  }
+
+  onAcknowledge() {
+    this.props.dispatch(ac.SetPref(this.props.disclaimerPref, false));
+    this.props.dispatch(ac.UserEvent({event: at.SECTION_DISCLAIMER_ACKNOWLEDGED, source: this.props.eventSource}));
```

Comment:
Looks like this is the only place, where `at.SECTION_DISCLAIMER_ACKNOWLEDGED` gets called. You don't need to create a new action type for this, a regular string `"SECTION_DISCLAIMER_ACKNOWLEDGED"` should just be fine.